test/test_resign.py

```diff
@@ -2,12 +2,44 @@
 
 from conftest import have_faketime
 from helper import run
+from decode_cms import decode_cms
+
+
+# each signer cert's issuer + serial
+SIGNERS = {
+    "release-1": ("O=Test Org, CN=Test Org Provisioning CA Release", 1),
+    "release-2018": ("O=Test Org, CN=Test Org Provisioning CA Release", 2),
+    "autobuilder-1": ("O=Test Org, CN=Test Org Provisioning CA Development", 1),
+    "autobuilder-2": ("O=Test Org, CN=Test Org Provisioning CA Development", 2),
+}
+
+
+def get_cms(bundle_path):
+    with open(bundle_path, "rb") as bundle_file:
+        bundle_file.seek(-8, 2)
+        cms_length_bytes = bundle_file.read(8)
+        cms_length = int.from_bytes(cms_length_bytes, "big")
+        cms_start = bundle_file.tell() - cms_length - 8
+        bundle_file.seek(cms_start)
+        cms_data = bundle_file.read(cms_length)
+
+    return decode_cms(cms_data)
+
+
+def get_signers(bundle_path):
+    cms = get_cms(bundle_path)
+    assert cms["contentType"] == "signedData"
+
+    cms_signers = {(si["issuer"], si["serial"]) for si in cms["signerInfos"]}
```

Comment:
Maybe add a short note that this already prepares for multiple signers? Since in `SIGNERS` (and in general) we only have single signers and the `{}` wrapping looks a bit artificial otherwise.